### PR TITLE
13 add header

### PR DESCRIPTION
**Pull request by AntoineCAU**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -3,16 +3,14 @@ import './css/Header.scss';
 const Header = () => {
   return (
     <header className="Header">
-      <img src="/img/logoDesktop.svg" alt="logo" className="logoDesktop" />
+      <img src="/img/logoPokemonDesk.png" alt="logo" className="logoDesktop" />
```
Comment (gasparddhi) [inline src/components/Header.jsx:6]: Love this new logo

**Comment by gasparddhi** [comment]
Everything is correct for me

**Comment by LoicBrassart** [comment]
Gaspard needs to validate the PR before assigning me, plz ;-)

**Review by gasparddhi — APPROVED** [review]
Everything correct

File: src/index.scss [hunk 2]
```diff
@@ -5,4 +5,5 @@
   padding: 0;
   box-sizing: border-box;
   font-family: 'Raleway', sans-serif;
-}
\ No newline at end of file
+}
+
```
Comment (LoicBrassart) [inline src/index.scss]: That's the exact kind of change I don't want to see anymore... Is anyone in your group working without Prettier/ESLint ?
If I see this again, I'll close PRs without discussion.

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil (except a useless change, prettier-related I thnk)

**Pull request merged**